share/cmake/modules/FindExtPackages.cmake

```diff
@@ -197,13 +197,10 @@ if((OCIO_BUILD_APPS AND OCIO_USE_OIIO_FOR_APPS) OR OCIO_BUILD_TESTS)
     # Supported from OIIO 2.4+. Setting this for lower versions doesn't affect anything.
     set(OPENIMAGEIO_CONFIG_DO_NOT_FIND_IMATH 1)
 
-    include(ocio_check_dependency_version)
-    # Since OpenImageIO will try to find OpenEXR through its OpenImageIOConfig.cmake file,
-    # let's try to find OpenEXR first and if the version is too old, OCIO will not try to find
-    # OpenImageIO.
-    ocio_check_dependency_version(  OpenEXR "is_OpenEXR_VERSION_valid" 
-                                    MIN_VERSION ${OpenEXR_MININUM_VERSION} 
-                                    CONFIG)
+    set(is_OpenEXR_VERSION_valid FALSE)
+    # Check for compatibility between OpenEXR and OpenImageIO.
+    # Will set is_OpenEXR_VERSION_valid to TRUE if valid.
+    include(CheckForOpenEXRCompatibility)
```

Comment:
Changed how we check the version of the OpenEXR used by OpenImageIO. This should be more robust and won't create any unwanted target or variables.